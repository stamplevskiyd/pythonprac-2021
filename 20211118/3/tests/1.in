try:
    alp = Alpha(c=10, z=2, a=42)
    print(alp.e)
except AttributeError:
    print("Alpha attribute error works")

try:
    alp = AlphaQ(c=10, z=2, a=42)
    print(alp.e)
except AttributeError:
    print("AlphaQ attribute error works")
    
