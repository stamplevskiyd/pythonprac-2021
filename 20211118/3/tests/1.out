Alpha attribute error works
AlphaQ attribute error works
